Mathlib/Order/SetNotation.lean

```diff
@@ -0,0 +1,302 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Patrick Massot
+-/
+import Mathlib.Data.Set.Defs
+import Mathlib.Mathport.Notation
+
+/-!
+# Notation classes for set supremum and infimum
+
+In this file we introduce notation for indexed suprema, infima, unions, and intersections.
+
+## Main definitions
+
+- `SupSet α`: typeclass introducing the operation `SupSet.sSup` (exported to the root namespace);
+  `sSup s` is the supremum of the set `s`;
+- `InfSet`: similar typeclass for infimum of a set;
+- `iSup f`, `iInf f`: supremum and infimum of an indexed family of elements,
+  defined as `sSup (Set.range f)` and `sInf (Set.range f)`, respectively;
+- `Set.sUnion s`, `Set.sInter s`: same as `sSup s` and `sInf s`,
+  but works only for sets of sets;
+- `Set.iUnion s`, `Set.iInter s`: same as `iSup s` and `iInf s`,
+  but works only for indexed families of sets.
+
+## Notation
+
+- `⨆ i, f i`, `⨅ i, f i`: supremum and infimum of an indexed family, respectively;
+- `⋃₀ s`, `⋂₀ s`: union and intersection of a set of sets;
+- `⋃ i, s i`, `⋂ i, s i`: union and intersection of an indexed family of sets.
+
+-/
```

Comment:
Where did this module doc come from?  Shouldn't you be one of the authors?  :)

Reply:
Done.